### DISCUSSION TITLE
Compute motif accessibility per cell

### DISCUSSION THREAD

**Discussion by biren-dave** [stuart-lab/signac#1184] [Q&A] [answered]
Hello,

I was wondering if it is possible to compute the accessibility of a given motif for each cell within a Signac object. 

I am able to identify enriched motifs after performing differential accessibility analysis, but cannot seem to find a way to get at the above.

Thanks for your help,

Biren

**Comment by timoast** [comment, marked as answer]
You can use chromVar for this
